parser_test.go

```diff
@@ -3076,11 +3122,11 @@ func TestSelectStatement(t *testing.T) {
 							From: &AliasedTableExpr{
 								Expr: &Table{Name: "t2", IsTarget: true},
 							},
-						},
-						OrderBy: []*OrderingTerm{
-							{
-								Expr:      &Column{Name: "a"},
-								Direction: AscStr,
+							OrderBy: []*OrderingTerm{
```

Comment:
The `OrderBy` is part of the right hand `Select` now, and not part of the `CompondSelect`